src/targets/gpu/fuse_mlir.cpp

```diff
@@ -61,12 +62,46 @@ bool mlir_enabled()
 #endif
 }
 
-static bool is_requested(std::string_view option, bool fallback = false)
+namespace {
+struct requested
+{
+};
+struct rejected
+{
+};
+} // namespace
+
+template <class Action>
+static std::vector<std::string> get_usage()
+{
+    static const auto options =
+        split_string(string_value_of(MIGRAPHX_MLIR_USE_SPECIFIC_OPS{}, ""), ',');
+    static const auto enabled = std::is_same<Action, requested>{};
+    std::vector<std::string> result;
+    auto remove_not_symbol = [&](const std::string& s) {
+        if(starts_with(s, "!"))
```

Comment:
Do you plan to add `~` symbol as well?

Reply:
so MIGRAPHX_MLIR_USE_SPECIFIC_OPS="!attention,dot" would not do any attentions through MLIR but would use MLIR for all dots ?